src/60_Samples_%26_Templates/Housing.html

```diff
@@ -202,41 +233,41 @@ <h4>Floorplan</h4>
         </section>
         <section>
           <h4>Mortgage Calculator</h4>
-          <form class="px1 pt1" method="GET" action="/samples_templates/housing/calculate-mortgage" action-xhr="/samples_templates/housing/calculate-mortgage-xhr"  target="_top">
-            <div class="flex mb1">
-              <span class="form-field-left">
+          <form method="GET" action="/samples_templates/housing/calculate-mortgage" action-xhr="/samples_templates/housing/calculate-mortgage-xhr"  target="_top">
+            <div>
```

Comment:
Can you simplify the markup here (remove unused spans and divs). Can you also use `label` for the input field description. I think it's fine for label and input to be on the same line.

Reply:
done